core/modules/ckeditor5/lib/UPGRADING.md

```diff
@@ -1,46 +1,33 @@
 # Updating CKEditor 5
 
-This directory contains a custom DLL build of CKEditor 5. Normal builds of
-CKEditor, such as the one created through the CKEditor Online Builder Tool, must
-be configured with desired plugins at the time it is compiled. Because Backdrop
-needs to allow plugins to be loaded by individual modules, using the normal
-builder tool is not suitable.
-
-A DLL (Dynamically Linked Library) build on the other hand allows plugins to
-be loaded as separate individual files. When updating CKEditor 5, a new DLL
-build must be created.
-
-See https://ckeditor.com/docs/ckeditor5/latest/installation/advanced/alternative-setups/dll-builds.html#creating-a-dll-build
+This directory contains CKEditor 5 UMD build, available via npm.
 
 ## Steps to update
 
 1. Install npm if you do not have it on your system. The recommended way to get
    npm running is to install nvm (Node Version Manager), which in turn installs
    npm and allows you to switch between multiple different versions.
-2. Make a temporary working directory anywhere on your system. e.g.
+2. Download the latest version
    ```
-   mkdir ~/ckeditor5-temp
-   cd ~/ckeditor5-temp
+   npm install --save ckeditor5
    ```
-3. Copy the script from lib/scripts there and make it executable:
+3. Copy relevant umd files (and only those) from translations directory:
    ```
-   cp [original/location/]concat-build.sh concat-build.sh
-   chmod +x concat-build.sh
+   cp node_modules/ckeditor5/dist/translations/*umd.js PATH/TO/core/modules/ckeditor5/lib/ckeditor5/dist/translations/
    ```
-4. Run the script. This will download the DLL build via npm and concat the core
-   and plugin builds into one file.
+4. Copy the relevant files to the dist folder of your dev branch
    ```
-   ./concat-build.sh
+   cp node_modules/ckeditor5/dist/browser/ckeditor5.umd.js PATH/TO/core/modules/ckeditor5/lib/ckeditor5/dist/browser/
+   cp node_modules/ckeditor5/dist/browser/ckeditor5.css PATH/TO/core/modules/ckeditor5/lib/ckeditor5/dist/browser/
```

Comment:
Let's add a line/step to delete the sourcemap out of the `ckeditor5.umd.js` file.